clang/test/CIR/CodeGen/switch.cpp

```diff
@@ -300,3 +333,48 @@ void fallthrough(int x) {
 // CHECK-NEXT:      }
 // CHECK-NEXT:      ]
 // CHECK-NEXT:    }
+
+int caseNone1(int a) {
+  switch (a)
+    return -1;
+  return 1;
+}
+
+//      CHECK:  cir.func @_Z9caseNone1i
+//      CHECK:    cir.scope {
+//      CHECK:      cir.switch
+// CHECK-NEXT:      case (none) {
```

Comment:
Given the body of the switch is never going to executed, instead of introducing `none`, we should just emit an empty switch body instead.